internal/command/machine/launch.go

```diff
@@ -168,7 +168,7 @@ func run(ctx context.Context) (err error) {
 		return
 	}
 
-	appConfig.WriteToDisk()
+	//	appConfig.WriteToDisk()
```

Comment:
what is going on here?

Reply:
This is an aborted port of the launch command to the machines platform. This command is not visible to users, but we could remove it and revive later if desired.